travis-build.bash

```diff
@@ -20,6 +20,11 @@ function site-build() {
         return 1
     fi
     
+    # At one point our CSS files got in with the wrong metadata
+    # and now they are screwy in caches across the interwebs.
+    # Give them all different names than they had that day.
+    ./rename-css.sh
```

Comment:
You do not check the exit value of this. If it fails, do you still want the build to proceed?